src/Sylius/Bundle/ResourceBundle/DependencyInjection/Extension/AbstractResourceExtension.php

```diff
@@ -179,6 +179,7 @@ protected function registerFormTypes(array $config, ContainerBuilder $container)
                     $definition->setArguments(array(
                         $serviceClasses['model'],
                         new Parameter(sprintf('%s.validation_group.%s%s', $this->applicationName, $model, $suffix)),
+                        $this->applicationName,
```

Comment:
How to put bundle underscored name instead of ```sylius``` only.
For example: ```SyliusOrderBundle -> sylius_order```, ```SyliusProductBundle -> sylius_product``` etc ?